gflownet/envs/cube.py

```diff
@@ -869,10 +934,12 @@ def _get_logprobs_forward(
         logprobs_increments_rel = torch.zeros(
             (n_states, self.n_dim), dtype=self.float, device=self.device
         )
-        jacobian_diag = torch.ones(
+        log_jacobian_diag = torch.zeros(
             (n_states, self.n_dim), device=self.device, dtype=self.float
         )
         eos_tensor = tfloat(self.eos, float_type=self.float, device=self.device)
+        # Mask of effective dimensions
+        is_effective_dim = ~mask[:, -self.n_dim :]
```

Comment:
Same here for this variable which seems unused.

Reply:
https://github.com/alexhernandezgarcia/gflownet/pull/209/commits/d6303ec7490aa35e74213adfb7346f4b315b39ea